lib/components/component.dart

```diff
@@ -100,6 +111,22 @@ abstract class PositionComponent extends Component {
     final double dx = -anchor.relativePosition.dx * width;
     final double dy = -anchor.relativePosition.dy * height;
     canvas.translate(dx, dy);
+
+    if (debugMode) {
+      canvas.drawRect(Rect.fromLTWH(0.0, 0.0, width, height), _debugPaint);
```

Comment:
extract to a renderDebugMode method so that it can be overwritten easily by custom components